src/client.js

```diff
@@ -906,6 +901,10 @@ export class StreamChat {
 			...options,
 		};
 
+		if (!this._hasConnectionID()) {
+			payload.watch = false;
```

Comment:
we also need to handle
payload.presence = false

Reply:
presence also requires connection_id

Reply:
Done!